airbyte-webapp/src/components/StatusIcon/StatusIcon.tsx

```diff
@@ -23,29 +24,32 @@ const _iconByStatus: Partial<Record<StatusIconStatus, IconDefinition | undefined
   warning: faExclamationTriangle,
 };
 
-const _themeByStatus: Record<StatusIconStatus, string> = {
+const _themeByStatus: Partial<Record<StatusIconStatus, string>> = {
   empty: "attentionColor",
   inactive: "lightTextColor",
   success: "successColor",
   warning: "warningColor",
 };
 
-const Badge = styled.div<Props>`
+const Container = styled.div<Props>`
   width: ${(props) => getBadgeWidth(props)}px;
   height: ${({ big }) => (big ? 40 : 20)}px;
-  background: ${(props) => props.theme[(props.status && _themeByStatus[props.status]) || "dangerColor"]};
-  box-shadow: 0 1px 2px ${({ theme }) => theme.shadowColor};
-  border-radius: ${({ value }) => (value ? "15px" : "50%")};
   margin-right: 10px;
-  padding-top: 4px;
-  color: ${({ theme }) => theme.whiteColor};
   font-size: ${({ big }) => (big ? 24 : 12)}px;
   line-height: ${({ big }) => (big ? 33 : 12)}px;
   text-align: center;
   display: inline-block;
   vertical-align: top;
 `;
 
+const Badge = styled(Container)<Props>`
+  background: ${(props) => props.theme[(props.status && _themeByStatus[props.status]) || "dangerColor"]};
+  box-shadow: 0 1px 2px ${({ theme }) => theme.shadowColor};
```

Comment:
Maybe we could remove the box-shadow here. It feels also looking a bit "old-fashion" in this place, and I think it would look a bit better if we get rid of it similar to the connector icons. What do you think?

Reply:
I reviewed the design and it does indeed not have a shadow. I will remove.